Tests/UnitTests/SubscriberAttributes/BackendSubscriberAttributesTests.swift

```diff
@@ -78,8 +78,6 @@ class BackendSubscriberAttributesTests: TestCase {
     // MARK: PostReceipt with subscriberAttributes
 
     func testPostReceiptWithSubscriberAttributesSendsThemCorrectly() throws {
-        var completionCallCount = 0
```

Comment:
Hmm so we were not verifying this right? Instead of removing it, should we verify it instead?

Reply:
No haha. Yeah I can add it back

Reply:
Actually it's checked by other tests already, so I think it's better to keep this test focused on testing this one thing.